packages/rollup-plugin-html/README.md

```diff
@@ -73,7 +74,61 @@ export default {
 
 </details>
 
-You can also set the `inputPath` property on the html plugin. This is useful if you are generating multiple html files, which each have their own entrypoints:
+#### Input from multiple html files
+
+You can also work with multiple html files. This can be the case for example if a static site generator is used.
+In order to get all html files you can provide a glob or array of globs as an input.
+Be sure to specify `flatten: false` for the plugin to retain the folder structure.
+
+<details>
+
+<summary>Show example</summary>
+
+```js
+import html from '@open-wc/rollup-plugin-html';
+export default {
+  input: '**/*.html',
+  // or even
+  input: ['index.html', 'pages/*.html'],
+  output: { dir: 'dist' },
+  plugins: [html({ flatten: false })],
+};
+```
+
+</details>
+
+When keeping the folder structures it may mean that your `rootDir` is not your current working directory.
+Image for example all the html files are generated in to a `_site-in-html` folder.
```

Comment:
I think it's better to explain this option in detail at the bottom, and to show just a code example here.

Reply:
moved it